src/theme/components/progressIndicator.ts

```diff
@@ -53,9 +53,13 @@ const ProgressIndicator = defineMultiStyleConfig({
         circularContainer: {
           alignItems: "center",
           display: "flex",
-          flexDirection: "column",
+          flexDirection: labelPlacement === "bottom" ? "column" : "row",
           width: "fit-content",
         },
+        circularLabel: {
+          marginBottom: 0,
+          marginLeft: "xs",
```

Comment:
Since there is a marginLeft here, when the label is on the right, should there be a marginTop for then the label is below the indicator?